js/index.js

```diff
@@ -1,42 +1,75 @@
-// ITERATION 1
+//Global HTML selectors
+const displayTotal = document.querySelector('#total-value span')
+const itemList = document.querySelector('tbody')
+const newProductName = document.querySelector('.product-name input')
```

Comment:
As far as I can see, these variables are only used in one function. Thus, I would declare them over there.